packages/web-app-files/src/index.js

```diff
@@ -127,7 +127,14 @@ export default {
   userReady({ store }) {
     archiverService.initialize(
       store.getters.configuration.server || window.location.origin,
-      get(store, 'getters.capabilities.files.archivers', [])
+      get(store, 'getters.capabilities.files.archivers', [
```

Comment:
long term we should add the capability to `core` / php based backend. Added it as a fallback for compatibility in the meantime.

Reply:
True. Care to open a ticket in github.com/owncloud/core/issues?:)